src/Installer/components/Modal/index.jsx

```diff
@@ -137,7 +141,6 @@ const Modal = ( {
 				},
 			} );
 			setPluginStatus( 'completed' );
-			showModal( false );
```

Comment:
This might not close the modal if there is an internal redirect, If that is okay it should be fine.

For example we are on: `{siteUrl}/wp-admin/admin.php?page=bluehost#/home` and we install the **WonderCart Plugin** with the CTA: [DATA](https://github.com/newfold-labs/wp-module-solutions/blob/54366a325423896bed1381630171266633a387b0/includes/js/debug.json#L888C25-L888C94) It redirects us to an internal page and I am not sure if the modal closes in that case.